### PR TITLE
Array binary search

### PR DESCRIPTION
**Pull request by dawitz28**

### CODE REVIEW

**Comment by dawitz28** [comment]
- [ 

- [/ ] ] Top-level README “Table of Contents” is updated

 - [/ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [/ ] Expected failure
     - [/ ] Edge Case (if applicable/obvious)
 - [/ ] README for this challenge is complete
     - [/ ] Summary, Description, Approach & Efficiency, Solution
     - [/ ] Link to code
     - [/ ] Picture of whiteboard

File: javascript/code-challenges/arrayBinarySearch/_test_/array-binary.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const binarySearch = require('../arrayBinarySearch/array-binary-search.js')
+
+describe('test array-binary-search', () => {
+    it('should return index of the array if matched the search key'() => {
```
Comment (lizkavalski) [inline javascript/code-challenges/arrayBinarySearch/_test_/array-binary.test.js:6]: forgot a comma should be:
```
it('should return index of the array if matched the search key',() => {
```
